scripts/lint_site.sh

```diff
@@ -24,6 +24,13 @@ else
     LANGS="en zh"
 fi
 
+red='\e[0;31m'
+clr='\e[0m'
+
+error() {
+  echo -e "${red}$@${clr}"
```

Comment:
```suggestion
  echo -e "${red}$*${clr}"
```
to fix the lint error, and allow `make lint` to run.